### DISCUSSION TITLE
Can't use cmd or GUI version

### DISCUSSION THREAD

**Discussion by JAIvY93** [Girbons/comics-downloader#108] [General]
Hi there, 

I have just found your github project and was hoping to use it. I have downloaded both comics-downloader.exe and the comics-downloader-gui-windows.exe files and they both do not work for me. 

When I try and open the comics-downloader.exe the cmd opens for a second and then closes and nothing else happens. When I open the GUI and fill out the information and click download it just comes up with INFO:downloading and then the GUI closes. I don't think I should need to download anything else to get this running?

Any help you can provide would be much appreciated.

Cheers,

**Comment by Girbons** [comment]
Hi @JAIvY93, 

The comics-downloader.exe is a CLI executable and it's not supposed to be used with a double click,
regarding the gui version I suggest to open an issue and follow the template guidelines.

**Reply by JAIvY93** [reply]
Thanks for the reply, I have tried to run the comics-downloader.exe from the cmd but it exhibited the exact same problem of the cmd flashing on and off.

**Reply by Girbons** [reply]
Ok, please open a [bug report](https://github.com/Girbons/comics-downloader/issues/new/choose) you will find a template to fill 😄
